ddtrace/_trace/trace_handlers.py

```diff
@@ -734,11 +740,18 @@ def _on_botocore_bedrock_process_response(
             span.set_tag_str("bedrock.response.choices.{}.id".format(i), str(body["generations"][i]["id"]))
     integration = ctx["bedrock_integration"]
     if metadata is not None:
+        usage = {}  # catch usage for streamed response case
         for k, v in metadata.items():
             if k in ["usage.completion_tokens", "usage.prompt_tokens"] and v:
                 span.set_metric("bedrock.response.{}".format(k), int(v))
+                if k == "usage.completion_tokens":
+                    usage["output_tokens"] = int(v)
+                if k == "usage.prompt_tokens":
+                    usage["input_tokens"] = int(v)
             else:
                 span.set_tag_str("bedrock.{}".format(k), str(v))
+        if usage:
+            ctx.set_item("llmobs.usage", usage)
```

Comment:
I think for the purposes of llmobs-specific code, I'd prefer it to live in `ddtrace/contrib/internal/botocore/services/bedrock.py` rather than in the shared-product `trace_handlers.py` file. Can we move this logic to the streaming body extract logic instead?